### PR TITLE
Feature/jude/get and update profile

### PR DESCRIPTION
**Pull request by judeokafor**
Authenticated User can get his profile details
Authenticated user can update his profile details

### CODE REVIEW

File: functions/.eslintrc.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
 		"max-len": ["error", {
 			"code": 100
 		}],
-		"max-lines-per-function": ["error", 25],
+		"max-lines-per-function": ["error", 20],
```
Comment (Simpcyclassy) [inline functions/.eslintrc.json:85]: Are we going with 20 or 30, there seems to be a conflict between your choices?

**Review by Simpcyclassy — APPROVED** [review]
Looks good to me?

File: functions/.eslintrc.json (hunk 1 above)
Comment (judeokafor) [reply functions/.eslintrc.json:85]: I was checking out something and forgot to revert back to 20. So i just did now. Its 20 from the original spec given to us.

Comment (Simpcyclassy) [reply functions/.eslintrc.json:85]: OK cause I saw 30 in the previous PR and also in Faith's change

**Comment by Way2nnadi** [comment]
This looks good, just go through your endpoints and make sure your are [using the http-status-code library](https://github.com/enyeInc/Engineering-Conventions/blob/master/README.md#http-status-codes)

**Pull request merged**